### DISCUSSION TITLE
Restricted User profile and capabilities

### DISCUSSION THREAD

**Discussion by kranthikud** [netblue30/firejail#5520] [Q&A]
Added a new Restricted user(tool_debug) with the below details inside the` login.users` file

`tool_debug: --hostname=host_debug --name=fj_debug --net=eth0 --protocol=unix,inet,inet6,netlink,packet --caps.keep=net_bind_service,net_raw --noprofile`

The username is working and able to login the user. But there are couple of challenges.

1. What is the default profile file the Restricted user? Even though I have provided the --noprofile config, it is adding some more restrictions. Are there any restrictions for this user which we can not modify?
2. Why the socket operation is not permitted for tcpdump in Restricted user mode.
`[tool_debug@sp_host_debug ~]$ /usr/sbin/tcpdump -nnvvi eth0-28154
tcpdump: eth0-28154: You don't have permission to capture on that device
(socket: Operation not permitted)
`

**Comment by rusty-snake** [comment]
1. Which other restrictions do you face? Note that some restrictions are hardcoded and others implied by the commands you listed. 
2. Dies it work if you omit the protocol and hostname commands?

**Reply by kranthikud** [reply]
@rusty-snake,
It did not work even after omitting the protocol and hostname. It is still throwing the same error as above. I have tried the following 2 ways also. 
`tool_debug: --noprofile --caps.keep=net_bind_service,net_raw`
`tool_debug: --noprofile`
You have mentioned some restrictions are hardcoded. Can I get any document or point me to the code to understand the details. Thanks.

**Reply by rusty-snake** [reply]
I don't think that there is any good documentation about this, neither by firejail nor by the kernel.

Try it with `--profile=noprofile`.
